src/app/app.component.spec.ts

```diff
@@ -1,13 +1,32 @@
 import { TestBed, async } from '@angular/core/testing';
-
+import {HeaderComponent} from './header/header.component';
 import { AppComponent } from './app.component';
```

Comment:
Inconsistent spascing... I would say change all the import statements to Import { name } instead of import{name}